Library/Koala/Koala.swift

```diff
@@ -1591,6 +1591,20 @@ public final class Koala {
     self.track(event: "Changed Live Stream Orientation", properties: props)
   }
 
+  public func trackClosedLiveStream(project: Project,
```

Comment:
Ah right, I was thinking that it could be inferred from the timestamp that's tracked but probably best to just do a `duration_watched`.  I'll do a `TimeInterval` for that.